discord/interpreter.js

```diff
@@ -194,8 +213,8 @@ module.exports.handleCom = async (message, client) => {
                     Logger.log("info", `Could not find channel: ${channelName}`)
                 } else {
                     Logger.log("info", "Admin channel found. Sending message...")
-                    channel.send(`New user ${message.author.username} (real name apparently ${realName}, identification provided: ${identification}) has requested to join the server.
-If the user seems legitimate, please add them via \`!insertuser ${message.author.username}, ${realName}\``);
+                    channel.send(`New user ${message.author.username} (real name apparently ${realName}, fresher: ${fresher ? "yes" : "no"}, identification provided: ${identification}) has requested to join the server.
```

Comment:
`fresher: ${fresher ? "yes" : "no"},`

Technically you don't need this as you already check for null above so `fresher` will always have a value. If you keep `fresher` as a string then you can just insert the value of fresher.
`fresher: ${fresher}`